include/CL/opencl.hpp

```diff
@@ -6703,6 +6703,9 @@ class Program : public detail::Wrapper<cl_program>
 #if CL_HPP_TARGET_OPENCL_VERSION >= 120
     cl_int compile(
         const char* options = nullptr,
+        cl_uint num_input_headers = 0,
+        const cl_program* input_headers = nullptr,
+        const char** header_include_names = nullptr,
```

Comment:
I'm less sure about this one, but could this be a `vector` of `strings` instead?